rnd/market/Dockerfile

```diff
@@ -25,19 +26,31 @@ ENV POETRY_VERSION=1.8.3 \
     PATH="$POETRY_HOME/bin:$PATH"
 RUN pip3 install poetry
 
-COPY rnd/market /app/rnd/market
+FROM server_base AS server_dependencies
+
+RUN mkdir -p /app/autogpt
+RUN mkdir -p /app/forge
+RUN mkdir -p /app/rnd/autogpt_libs
+RUN mkdir -p /app/rnd/market
 COPY rnd/autogpt_libs /app/rnd/autogpt_libs
 
+COPY rnd/market/poetry.lock rnd/market/pyproject.toml /app/rnd/market/
+
 WORKDIR /app/rnd/market
 
-# Install dependencies
 RUN poetry install --no-interaction --no-ansi
 
+FROM server_dependencies AS server_prisma
+
+# Need the market/utils/partial_types.py
+COPY rnd/market /app/rnd/market
+
+COPY rnd/market/schema.prisma ./
 RUN poetry run prisma generate
 
-FROM server_base as server
+FROM server_prisma AS server
 
-ENV PORT=8000
+ENV PORT=8005
```

Comment:
IMO we should use a consistent default port for types of services, e.g. HTTP/REST servers = port 8000. We can set it to a different port in the docker compose config or the deploy config. Baking different default ports into the images will just cause confusion.

Reply:
Yeah fair point, I agree.